.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
```

Comment:
Add a section on environment and collect following data? 

- Hudi version :
- Spark version :
- Hive version :
- Hadoop version :
- Storage(HDFS/S3/GCS..) :

Reply:
And whether docker or not?

Reply:
Will it matter if a user is running on docker?

Reply:
@Guru107 IMO, collecting enough information for analyzing and discussing is always good. WDYT? It's only a boolean option.

Reply:
Cool! Makes sense